core/chains/evm/gas/models.go

```diff
@@ -329,7 +331,7 @@ func bumpDynamicFee(cfg Config, lggr logger.Logger, currentTipCap, currentBaseFe
 
 	if currentTipCap != nil {
 		if currentTipCap.Cmp(maxGasPrice) > 0 {
-			lggr.Errorf("invariant violation: ignoring current tip cap of %s that would exceed max gas price of %s", currentTipCap.String(), maxGasPrice.String())
+			lggr.Errorf("AssumptionViolation: Ignoring current tip cap of %s that would exceed max gas price of %s", currentTipCap.String(), maxGasPrice.String())
```

Comment:
I wonder if this would easier to use with some kind of helper. I don't want to expand the log interface with more methods (especially since it's not a proper level of its own), but perhaps we could use something like a standard `type AssumptionViolationError struct` that formats them all the same :shrug: